packages/charts/src/components/ScatterChart/ScatterChart.tsx

```diff
@@ -211,6 +216,7 @@ const ScatterChart: FC<ScatterChartProps> = forwardRef((props: ScatterChartProps
             tick={<XAxisTicks config={xMeasure} />}
             padding={xAxisPadding}
             height={xAxisHeights[0]}
+            reversed={useIsRTL()}
```

Comment:
please don't call hooks inside props. extract it to `const isRTL = useIsRTL()` somewhere before the return and just use the value here.